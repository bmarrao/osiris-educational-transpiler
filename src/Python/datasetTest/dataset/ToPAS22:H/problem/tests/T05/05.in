5
4 5
A B C D
C B
C D
C A
A D
A B
4 5
A B C D
C D
D A
C A
C B
D B
4 5
A B C D
C D
A B
A D
C A
C B
4 5
A B C D
C A
C D
B D
B A
C B
4 5
A B C D
D B
B C
D A
B A
D C
