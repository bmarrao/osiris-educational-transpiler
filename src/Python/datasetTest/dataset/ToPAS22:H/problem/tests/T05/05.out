Impossivel
Impossivel
Impossivel
Impossivel
Impossivel
